src/main/java/com/personal/financeManager/documentProcessor/models/DataEntityBean.java

```diff
@@ -0,0 +1,30 @@
+package com.personal.financeManager.documentProcessor.models;
+
+import java.util.Map;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class DataEntityBean {
+	
+	@NotBlank(message = "featureID cannot be null or blank")
+	private String featureID;
+	
+	@NotBlank(message = "featureVariantID cannot be null or blank")
+	private String featureVariantID;
+	
+	@NotBlank(message = "datEntityID cannot be null or blank")
```

Comment:
dataEntityID is not a required field, so please check it and remove Not Blank validation